bundles/org.eclipse.ui.forms/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %name
 Bundle-SymbolicName: org.eclipse.ui.forms;singleton:=true
-Bundle-Version: 3.12.100.qualifier
+Bundle-Version: 3.13.000.qualifier
```

Comment:
Maybe the version has to be changed as follows.
```suggestion
Bundle-Version: 3.13.0.qualifier
```

Reply:
Absolutely!

Reply:
thanks